aten/src/ATen/native/native_functions.yaml

```diff
@@ -4238,13 +4238,12 @@
   dispatch:
     CPU, CUDA: tanh_out
 
+# Alias to linalg.tensordot
```

Comment:
Adding an alias requires a few steps described in `# Note [Adding an alias]` in the same file `native_functions.yaml`.
Step 6 is missing.

Reply:
thanks, fixed by 7e7cb83239